{
  error: MongooseError: Connection.prototype.close() no longer accepts a callback
      at NativeConnection.close (C:\Projet perso\Convoiturage-Backend2\node_modules\mongoose\lib\connection.js:856:11)
      at Server.<anonymous> (C:\Projet perso\Convoiturage-Backend2\server.js:237:29)
      at Object.onceWrapper (node:events:632:28)
      at Server.emit (node:events:518:28)
      at emitCloseNT (node:net:2418:8)
      at process.processTicksAndRejections (node:internal/process/task_queues:89:21),
  level: 'error',
  message: 'unhandledRejection: Connection.prototype.close() no longer accepts a callback\n' +
    'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  stack: 'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  rejection: true,
  date: 'Sat Aug 09 2025 16:49:02 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 1892,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend2',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend2\\server'
    ],
    memoryUsage: {
      rss: 70361088,
      heapTotal: 34893824,
      heapUsed: 31518928,
      external: 38947516,
      arrayBuffers: 36509959
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 16773.781 },
  trace: [
    {
      column: 11,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js',
      function: 'NativeConnection.close',
      line: 856,
      method: 'close',
      native: false
    },
    {
      column: 29,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\server.js',
      function: null,
      line: 237,
      method: null,
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Object.onceWrapper',
      line: 632,
      method: 'onceWrapper',
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Server.emit',
      line: 518,
      method: 'emit',
      native: false
    },
    {
      column: 8,
      file: 'node:net',
      function: 'emitCloseNT',
      line: 2418,
      method: null,
      native: false
    },
    {
      column: 21,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 89,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-09 16:49:02'
}
{
  error: MongooseError: Connection.prototype.close() no longer accepts a callback
      at NativeConnection.close (C:\Projet perso\Convoiturage-Backend2\node_modules\mongoose\lib\connection.js:856:11)
      at Server.<anonymous> (C:\Projet perso\Convoiturage-Backend2\server.js:237:29)
      at Object.onceWrapper (node:events:632:28)
      at Server.emit (node:events:518:28)
      at emitCloseNT (node:net:2418:8)
      at process.processTicksAndRejections (node:internal/process/task_queues:89:21),
  level: 'error',
  message: 'unhandledRejection: Connection.prototype.close() no longer accepts a callback\n' +
    'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  stack: 'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  rejection: true,
  date: 'Sat Aug 09 2025 17:08:37 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 10852,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend2',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend2\\server'
    ],
    memoryUsage: {
      rss: 78131200,
      heapTotal: 34631680,
      heapUsed: 31527840,
      external: 38947516,
      arrayBuffers: 36509959
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 17948.593 },
  trace: [
    {
      column: 11,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js',
      function: 'NativeConnection.close',
      line: 856,
      method: 'close',
      native: false
    },
    {
      column: 29,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\server.js',
      function: null,
      line: 237,
      method: null,
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Object.onceWrapper',
      line: 632,
      method: 'onceWrapper',
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Server.emit',
      line: 518,
      method: 'emit',
      native: false
    },
    {
      column: 8,
      file: 'node:net',
      function: 'emitCloseNT',
      line: 2418,
      method: null,
      native: false
    },
    {
      column: 21,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 89,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-09 17:08:37'
}
{
  error: MongooseError: Connection.prototype.close() no longer accepts a callback
      at NativeConnection.close (C:\Projet perso\Convoiturage-Backend2\node_modules\mongoose\lib\connection.js:856:11)
      at Server.<anonymous> (C:\Projet perso\Convoiturage-Backend2\server.js:237:29)
      at Object.onceWrapper (node:events:632:28)
      at Server.emit (node:events:518:28)
      at emitCloseNT (node:net:2418:8)
      at process.processTicksAndRejections (node:internal/process/task_queues:89:21),
  level: 'error',
  message: 'unhandledRejection: Connection.prototype.close() no longer accepts a callback\n' +
    'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  stack: 'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  rejection: true,
  date: 'Sat Aug 09 2025 17:33:20 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 11596,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend2',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend2\\server'
    ],
    memoryUsage: {
      rss: 71065600,
      heapTotal: 35155968,
      heapUsed: 31531312,
      external: 38947516,
      arrayBuffers: 36509959
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 19430.953 },
  trace: [
    {
      column: 11,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js',
      function: 'NativeConnection.close',
      line: 856,
      method: 'close',
      native: false
    },
    {
      column: 29,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\server.js',
      function: null,
      line: 237,
      method: null,
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Object.onceWrapper',
      line: 632,
      method: 'onceWrapper',
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Server.emit',
      line: 518,
      method: 'emit',
      native: false
    },
    {
      column: 8,
      file: 'node:net',
      function: 'emitCloseNT',
      line: 2418,
      method: null,
      native: false
    },
    {
      column: 21,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 89,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-09 17:33:20'
}
{
  error: MongooseError: Connection.prototype.close() no longer accepts a callback
      at NativeConnection.close (C:\Projet perso\Convoiturage-Backend2\node_modules\mongoose\lib\connection.js:856:11)
      at Server.<anonymous> (C:\Projet perso\Convoiturage-Backend2\server.js:237:29)
      at Object.onceWrapper (node:events:632:28)
      at Server.emit (node:events:518:28)
      at emitCloseNT (node:net:2418:8)
      at process.processTicksAndRejections (node:internal/process/task_queues:89:21),
  level: 'error',
  message: 'unhandledRejection: Connection.prototype.close() no longer accepts a callback\n' +
    'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  stack: 'MongooseError: Connection.prototype.close() no longer accepts a callback\n' +
    '    at NativeConnection.close (C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js:856:11)\n' +
    '    at Server.<anonymous> (C:\\Projet perso\\Convoiturage-Backend2\\server.js:237:29)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Server.emit (node:events:518:28)\n' +
    '    at emitCloseNT (node:net:2418:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)',
  rejection: true,
  date: 'Sat Aug 09 2025 17:46:54 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 2316,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend2',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend2\\server'
    ],
    memoryUsage: {
      rss: 70549504,
      heapTotal: 34631680,
      heapUsed: 31511736,
      external: 38947516,
      arrayBuffers: 36509959
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 20244.968 },
  trace: [
    {
      column: 11,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\node_modules\\mongoose\\lib\\connection.js',
      function: 'NativeConnection.close',
      line: 856,
      method: 'close',
      native: false
    },
    {
      column: 29,
      file: 'C:\\Projet perso\\Convoiturage-Backend2\\server.js',
      function: null,
      line: 237,
      method: null,
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Object.onceWrapper',
      line: 632,
      method: 'onceWrapper',
      native: false
    },
    {
      column: 28,
      file: 'node:events',
      function: 'Server.emit',
      line: 518,
      method: 'emit',
      native: false
    },
    {
      column: 8,
      file: 'node:net',
      function: 'emitCloseNT',
      line: 2418,
      method: null,
      native: false
    },
    {
      column: 21,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 89,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-09 17:46:54'
}
{
  error: ReferenceError: next is not defined
      at inscription (C:\Projet perso\Convoiturage-Backend\controllers\authController.js:95:7)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5),
  level: 'error',
  message: 'unhandledRejection: next is not defined\n' +
    'ReferenceError: next is not defined\n' +
    '    at inscription (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:95:7)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  stack: 'ReferenceError: next is not defined\n' +
    '    at inscription (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:95:7)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  rejection: true,
  date: 'Tue Aug 12 2025 11:59:26 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 10148,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend\\server.js'
    ],
    memoryUsage: {
      rss: 91811840,
      heapTotal: 47108096,
      heapUsed: 43787856,
      external: 39151284,
      arrayBuffers: 36691453
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 16332.093 },
  trace: [
    {
      column: 7,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js',
      function: 'inscription',
      line: 95,
      method: null,
      native: false
    },
    {
      column: 5,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 105,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-12 11:59:26'
}
{
  error: ReferenceError: next is not defined
      at inscription (C:\Projet perso\Convoiturage-Backend\controllers\authController.js:95:7)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5),
  level: 'error',
  message: 'unhandledRejection: next is not defined\n' +
    'ReferenceError: next is not defined\n' +
    '    at inscription (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:95:7)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  stack: 'ReferenceError: next is not defined\n' +
    '    at inscription (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:95:7)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  rejection: true,
  date: 'Tue Aug 12 2025 12:00:01 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 16436,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend\\server.js'
    ],
    memoryUsage: {
      rss: 91615232,
      heapTotal: 50253824,
      heapUsed: 43396736,
      external: 39089549,
      arrayBuffers: 36629718
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 16366.953 },
  trace: [
    {
      column: 7,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js',
      function: 'inscription',
      line: 95,
      method: null,
      native: false
    },
    {
      column: 5,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 105,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-12 12:00:01'
}
{
  error: ReferenceError: next is not defined
      at inscription (C:\Projet perso\Convoiturage-Backend\controllers\authController.js:95:7)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5),
  level: 'error',
  message: 'unhandledRejection: next is not defined\n' +
    'ReferenceError: next is not defined\n' +
    '    at inscription (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:95:7)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  stack: 'ReferenceError: next is not defined\n' +
    '    at inscription (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:95:7)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  rejection: true,
  date: 'Tue Aug 12 2025 12:00:58 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 9576,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend\\server'
    ],
    memoryUsage: {
      rss: 91000832,
      heapTotal: 47108096,
      heapUsed: 43659328,
      external: 39103275,
      arrayBuffers: 36643444
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 16424.046 },
  trace: [
    {
      column: 7,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js',
      function: 'inscription',
      line: 95,
      method: null,
      native: false
    },
    {
      column: 5,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 105,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-12 12:00:58'
}
{
  error: ReferenceError: next is not defined
      at connexion (C:\Projet perso\Convoiturage-Backend\controllers\authController.js:207:5),
  level: 'error',
  message: 'unhandledRejection: next is not defined\n' +
    'ReferenceError: next is not defined\n' +
    '    at connexion (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:207:5)',
  stack: 'ReferenceError: next is not defined\n' +
    '    at connexion (C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js:207:5)',
  rejection: true,
  date: 'Tue Aug 12 2025 16:44:18 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 3956,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend\\server'
    ],
    memoryUsage: {
      rss: 105017344,
      heapTotal: 50778112,
      heapUsed: 48058288,
      external: 40378370,
      arrayBuffers: 37918539
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 33423.687 },
  trace: [
    {
      column: 5,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\controllers\\authController.js',
      function: 'connexion',
      line: 207,
      method: null,
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-12 16:44:18'
}
{
  error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
      at ServerResponse.setHeader (node:_http_outgoing:699:11)
      at ServerResponse.header (C:\Projet perso\Convoiturage-Backend\node_modules\express\lib\response.js:794:10)
      at ServerResponse.send (C:\Projet perso\Convoiturage-Backend\node_modules\express\lib\response.js:174:12)
      at ServerResponse.json (C:\Projet perso\Convoiturage-Backend\node_modules\express\lib\response.js:278:15)
      at C:\Projet perso\Convoiturage-Backend\middlewares\roleMiddleware.js:215:30
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
    code: 'ERR_HTTP_HEADERS_SENT'
  },
  level: 'error',
  message: 'unhandledRejection: Cannot set headers after they are sent to the client\n' +
    'Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n' +
    '    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n' +
    '    at ServerResponse.header (C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js:794:10)\n' +
    '    at ServerResponse.send (C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js:174:12)\n' +
    '    at ServerResponse.json (C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js:278:15)\n' +
    '    at C:\\Projet perso\\Convoiturage-Backend\\middlewares\\roleMiddleware.js:215:30\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  stack: 'Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n' +
    '    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n' +
    '    at ServerResponse.header (C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js:794:10)\n' +
    '    at ServerResponse.send (C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js:174:12)\n' +
    '    at ServerResponse.json (C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js:278:15)\n' +
    '    at C:\\Projet perso\\Convoiturage-Backend\\middlewares\\roleMiddleware.js:215:30\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)',
  rejection: true,
  date: 'Sun Aug 24 2025 23:46:17 GMT+0000 (temps universel coordonné)',
  process: {
    pid: 14016,
    uid: null,
    gid: null,
    cwd: 'C:\\Projet perso\\Convoiturage-Backend',
    execPath: 'C:\\Program Files\\nodejs\\node.exe',
    version: 'v22.15.0',
    argv: [
      'C:\\Program Files\\nodejs\\node.exe',
      'C:\\Projet perso\\Convoiturage-Backend\\server'
    ],
    memoryUsage: {
      rss: 92000256,
      heapTotal: 47632384,
      heapUsed: 45051704,
      external: 39349658,
      arrayBuffers: 36881317
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 14285.968 },
  trace: [
    {
      column: 11,
      file: 'node:_http_outgoing',
      function: 'ServerResponse.setHeader',
      line: 699,
      method: 'setHeader',
      native: false
    },
    {
      column: 10,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js',
      function: 'ServerResponse.header',
      line: 794,
      method: 'header',
      native: false
    },
    {
      column: 12,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js',
      function: 'ServerResponse.send',
      line: 174,
      method: 'send',
      native: false
    },
    {
      column: 15,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\node_modules\\express\\lib\\response.js',
      function: 'ServerResponse.json',
      line: 278,
      method: 'json',
      native: false
    },
    {
      column: 30,
      file: 'C:\\Projet perso\\Convoiturage-Backend\\middlewares\\roleMiddleware.js',
      function: null,
      line: 215,
      method: null,
      native: false
    },
    {
      column: 5,
      file: 'node:internal/process/task_queues',
      function: 'process.processTicksAndRejections',
      line: 105,
      method: 'processTicksAndRejections',
      native: false
    }
  ],
  service: 'covoiturage-api',
  environment: 'development',
  version: '1.0.0',
  timestamp: '2025-08-24 23:46:17'
}
